packages/connect/src/types/device.ts

```diff
@@ -34,6 +34,20 @@ export enum FirmwareType {
 // key = coin shortcut lowercase (ex: btc, eth, xrp) OR field declared in coins.json "supportedFirmware.capability"
 export type UnavailableCapabilities = { [key: string]: UnavailableCapability };
 
+export type FirmwareRevisionCheckResult =
+    | {
+          success: true;
+          error: null;
```

Comment:
btw isn't error: null kind of redundant?

Reply:
https://github.com/trezor/trezor-suite/pull/13495/commits/e410e009138e31a4ed6414cf5cf3203547ea272c